WinUIGallery/Package.WAP.Dev.appxmanifest

```diff
@@ -7,7 +7,7 @@
     xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
     IgnorableNamespaces="uap mp uap3">
   <!--Note: package version and assembly version must be kept in sync-->
-  <Identity Name="Microsoft.WinUI3ControlsGallery.Debug" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="2.1.0.0" />
+  <Identity Name="Microsoft.WinUI3ControlsGallery.Debug" Publisher="CN=Microsoft Corporation" Version="2.1.0.0" />
```

Comment:
>Publisher="CN=Microsoft Corporation" [](http://example.com/codeflow?start=57&length=36)

If this is being used for testing purposes, we could update the Publisher to something like "CN=WinUI-Gallery-Test" so it is clear that it is not an official build of something from Microsoft.